tests/e2e/dynamic_provisioning_test.go

```diff
@@ -142,6 +141,7 @@ var _ = Describe("CSI dynamic provisioning Test", func() {
 		fmt.Printf("PVC [%s/%s] updated size is [%v]\n", testNameSpaceName, testRetainPVCName, pvc.Spec.Resources.Requests.Storage())
 
 		By("PVC size should be updated")
+		// If the StorageClass storage profile name created is different from the cluster VM's storage profile name, csi-resizer may complain of VCD error.
```

Comment:
Could you also add the actual bugzilla there

Reply:
Added the bugzilla ID reference, thanks Arun!